### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Within settings dialog , User redirected to dialog not conveyed.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11249]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
https://sap.github.io/fundamental-ngx/#/platform/settings-generator. Within NGX platform settings , When the user activates 'Settings' UI and the user is redirected to a dialog, the same info must be conveyed by screen reader.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader easily can check NGX platform settings, by clicking setting dialog open for settings but same isn't conveyed by screen reader’.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by nikolay-kolarov** [comment]
Hi @JyotiKumari1507,

Could you please provide more info where you observed this?
I tested the last sample:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/f808a659-824d-43df-8d8d-870262a6d268)
And the dialog was correctly announced.

Thanks,
Nikolay

**Comment by JyotiKumari1507** [comment]
@nikolay-kolarov @droshev When the user activates the 'settings' UI, the user is redirected to a dialog, but the same info is not conveyed to the user by screen reader. please refer attached screenshot

<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/f5d4db7a-e13f-4e14-a00d-3b5a3015d0af">

**Comment by nikolay-kolarov** [comment]
Hi @droshev,
Hi @JyotiKumari1507,

As said, I tested the dialog on the provided test page and there the dialog role is announced.
So then the respective app can be checked if they use the same UI version, and the same component, and why it is not announced there.

Best Regards,
Nikolay

**Comment by JyotiKumari1507** [comment]
@droshev  @nikolay-kolarov  This issue may fix with NGX upgrade. If you want mean while i can get recording from accessibility team also.

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 can you create either stackblitz or share the steps how to reproduce the issue and what the expected behavior is?

**Comment by droshev** [comment]
@JyotiKumari1507 Can you create a stackblitz example, steps to reproduce and what the expectation is? Otherwise we will close the ticket.

**Comment by divyamalhotra488** [comment]
Hi @droshev, you can try with the example [here](https://stackblitz.com/edit/npoyvc-qnrzkh?file=package.json)

**Comment by droshev** [comment]
@divyamalhotra488 why can't you add whatever you need to the dialog template?

**Comment by droshev** [comment]
@divyamalhotra488 you have the control of the dialog, so you can set anything needed.

**Comment by divyamalhotra488** [comment]
@droshev, could you please suggest what attribute to set for screen reader to recognise dialog being opened ?

**Comment by droshev** [comment]
@divyamalhotra488 everything according to the spec or guideline.
